R/axis-secondary.R

```diff
@@ -213,30 +213,36 @@ AxisSecondary <- ggproto("AxisSecondary", NULL,
       range_info <- temp_scale$break_info()
 
       # Map the break values back to their correct position on the primary scale
-      old_val <- lapply(range_info$major_source, function(x) which.min(abs(full_range - x)))
-      old_val <- old_range[unlist(old_val)]
-      old_val_trans <- scale$trans$transform(old_val)
+      if (!is.null(range_info$major_source)) {
+        old_val <- lapply(range_info$major_source, function(x) which.min(abs(full_range - x)))
+        old_val <- old_range[unlist(old_val)]
+        old_val_trans <- scale$trans$transform(old_val)
 
-      old_val_minor <- lapply(range_info$minor_source, function(x) which.min(abs(full_range - x)))
-      old_val_minor <- old_range[unlist(old_val_minor)]
-      old_val_minor_trans <- scale$trans$transform(old_val_minor)
+        # rescale values from 0 to 1
+        range_info$major[] <- round(
+          rescale(
+            scale$map(old_val_trans, range(old_val_trans)),
+            from = range
+          ),
+          digits = 3
+        )
+      }
```

Comment:
Maybe we need a similar `else` branch here for `old_val_trans`?

Reply:
weird - I was sure I made it both places